### PR TITLE
French translation for CSS `:past` pseudo class reference

### PR DESCRIPTION
**Pull request by audrasjb**
This PR aims to create a French translation for CSS `:past` pseudo class reference.

### CODE REVIEW

File: files/fr/web/css/_colon_past/index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: ':past'
+slug: 'Web/CSS/:past'
+browser-compat: css.selectors.past
+translation_of: Web/CSS/:past
+---
+<p>{{CSSRef}}</p>
+
+<p>Le sélecteur de <a href="/fr/docs/Web/CSS/Pseudo-classes">pseudo-classe</a> <a href="/fr/docs/Web/CSS">CSS</a> <strong><code>:future</code></strong> est une pseudo-classe agissant dans la dimension temporelle qui cible n'importe quel élément apparaissant entièrement avant un élément correspondant à {{cssxref(":current")}}. Ce sélecteur peut par exemple servir dans le cas d'une vidéo ayant des sous-titres affichés à l'aide du format <a href="/fr/docs/Web/API/WebVTT_API">WebVTT</a>.</p>
```
Comment (SphinxKnight) [inline files/fr/web/css/_colon_past/index.html:9]: :future or :past Marty? :)

File: files/fr/web/css/_colon_past/index.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+<h3 id="html">HTML</h3>
+
+<pre class="brush: html">&lt;video controls preload="metadata"&gt;
+  &lt;source src="video.mp4" type="video/mp4" /&gt;
+  &lt;source src="video.webm" type="video/webm" /&gt;
+  &lt;track label="Français" kind="subtitles" srclang="fr" src="subtitles.vtt" default&gt;
+&lt;/video&gt;</pre>
+
+<h3 id="webvtt">WebVTT </h3>
```
Comment (SphinxKnight) [inline files/fr/web/css/_colon_past/index.html:35]: superfluous whitespace

File: files/fr/web/css/_colon_past/index.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+<p>{{Specifications}}</p>
+
+<h2 id="browser_compatibility">Compatibilité des navigateurs</h2>
+
+<p>{{Compat}}</p>
+
+<h2 id="see_also">Voir aussi</h2>
+
+<ul>
+	<li><a href="/fr/docs/Web/API/WebVTT_API">Web Video Text Tracks Format (WebVTT)</a></li>
```
Comment (SphinxKnight) [inline files/fr/web/css/_colon_past/index.html:63]: 2-spaces instead of tabs

**Review by SphinxKnight — CHANGES_REQUESTED** [review]

File: files/fr/web/css/_colon_past/index.html (hunk 3 above)
Comment (audrasjb) [reply files/fr/web/css/_colon_past/index.html]: AH! ok good to know.

Comment (audrasjb) [reply files/fr/web/css/_colon_past/index.html]: Fixed in https://github.com/mdn/translated-content/pull/1719/commits/dddffb1bbe97dbf47f03e8efb0a50c176da3b430

File: files/fr/web/css/_colon_past/index.html (hunk 2 above)
Comment (audrasjb) [reply files/fr/web/css/_colon_past/index.html]: Thanks, fixed in https://github.com/mdn/translated-content/pull/1719/commits/dddffb1bbe97dbf47f03e8efb0a50c176da3b430

File: files/fr/web/css/_colon_past/index.html (hunk 1 above)
Comment (audrasjb) [reply files/fr/web/css/_colon_past/index.html]: Oooops 🤭 Fixed in https://github.com/mdn/translated-content/pull/1719/commits/dddffb1bbe97dbf47f03e8efb0a50c176da3b430

**Review by SphinxKnight — APPROVED** [review]
Thanks :)

**Pull request merged**